packages/components/tests/dummy/app/templates/components/table.hbs

```diff
@@ -134,8 +156,9 @@
 </section>
 <section>
   <h3 class="dummy-h3" id="how-to-use"><a href="#how-to-use" class="dummy-link-section">§</a> How to use</h3>
-  <p class="dummy-paragraph">There are several ways to customize the table component.</p>
-  <h4 class="dummy-h4">Simple Table</h4>
+  <p class="dummy-paragraph">There are several ways to implement the table component. These examples will be in order of
+    increasing complexity. Additional implementation examples are in the Showcase section.</p>
```

Comment:
Notice: users/readers don't have (direct, easy) access to the code behind the examples in the showcase, so if we think for some of the examples the code should be exposed to the readers, we should include them as examples too in the showcase.